package.json

```diff
@@ -271,6 +271,7 @@
     "unified": "9.2.1"
   },
   "resolutions": {
+    "**/css-what": "^5.0.1",
```

Comment:
```suggestion
    "**/css-what": ">=5.0.1",
```

So we allow never major if required by other deps.

Reply:
Does it *allow* a newer major or *force* it? I was worried it would force an unnecessarily-high version later on.

Reply:
It should allow never, your resolution would force a downgrade to v5

Reply:
I think that the `>=` syntax forces the highest version. I tried `>=4.0.1` as an example:

![image](https://user-images.githubusercontent.com/6311784/121162093-e716cc00-c84d-11eb-8c90-60c8fd52ca30.png)

You can see it upgraded to 5.x unnecessarily, and I think it would upgrade to 6.x at the first unnecessary opportunity too.

Reply:
(Also note, I had deleted `node_modules` and `yarn.lock`)